tests/foreman/ui/test_products.py

```diff
@@ -217,7 +217,3 @@ def test_remove_prd(self):
                     )
                     self.assertIsNotNone(self.products.search(prd_name))
                     self.products.delete(prd_name)
-                    # Note: refresh is used here because sometimes selenium
-                    # is too fast to check the deleted object and it fails
-                    self.browser.refresh()
```

Comment:
Are you sure now this is taken care of? I remember the PR in which we specifically added browser refresh.